_data/publishers.csv

```diff
@@ -354,7 +354,6 @@ http://frdint.com/index.html,Fundamental Research and Development International,
 http://www.futureacademy.org.uk/,Future Academy,,
 http://www.gkpublication.in/,Galore Knowledge Publication Pvt. Ltd.,,
 http://www.gathacognition.com/,Gatha Cognition,,
-http://gavinpublishers.org/,Gavin Publishers,,
```

Comment:
Reviewed

Reply:
Please provide some evidence that the group is wrongly included in the list.
Responding with evidence to the contrary on the points listed here will suffice:
https://predatoryjournals.com/about/#some-basic-criteria